bot/exts/moderation/stream.py

```diff
@@ -0,0 +1,135 @@
+import time
+
+import discord
+from async_rediscache import RedisCache
+from discord.ext import commands, tasks
+
+from bot.bot import Bot
+from bot.constants import Guild, Roles, STAFF_ROLES, TIME_FORMATS
+
+# Constant error messages
+NO_USER_SPECIFIED = "Please specify a user"
+TIME_FORMAT_NOT_VALID = "Please specify a valid time format ex. 10h or 1day"
+TIME_LESS_EQ_0 = "Duration can not be a 0 or lower"
+USER_ALREADY_ALLOWED_TO_STREAM = "This user can already stream"
+USER_ALREADY_NOT_ALLOWED_TO_STREAM = "This user already can't stream"
+
+
+# FORMATS holds a combined list of all allowed time units
+# made from TIME_FORMATS constant
+FORMATS = []
+for key, entry in TIME_FORMATS.items():
+    FORMATS.extend(entry["aliases"])
+    FORMATS.append(key)
+
+
+class Stream(commands.Cog):
+    """Stream class handles giving screen sharing permission with commands."""
```

Comment:
I find it redundant to include the class's name in the docstring. I also don't think "class" should be included since 1. it's obvious it's a class, its declaration is one line above and 2. this docstring is user-facing via the help command. Here is a suggestion that's more in line with other cog docstrings:

```suggestion
    """Grant and revoke streaming permissions from users."""
```